src/mainnet/dca/job.ts

```diff
@@ -0,0 +1,76 @@
+import { Job, JobWorkableGroup, makeid, prelog, toKebabCase } from '@keep3r-network/cli-utils';
+import { Contract } from 'ethers';
+import DCAKeep3rJobABI from '../abi/DCAKeep3rJob.json';
+import metadata from './metadata.json';
+
+const jobAddress = '0xEcbA21E26466727d705d48cb0a8DE42B11767Bf7';
+
+const getWorkableTxs: Job['getWorkableTxs'] = async (args) => {
+  const correlationId = toKebabCase(metadata.name);
+  const logMetadata = {
+    job: metadata.name,
+    block: args.advancedBlock,
+    logId: makeid(5),
+  };
+  const logConsole = prelog(logMetadata);
+
+  if (args.skipIds.includes(correlationId)) {
+    logConsole.log(`Job in progress, avoid running`);
+    return args.subject.complete();
+  }
+
+  logConsole.log(`Trying to work`);
+
+  const job = new Contract(jobAddress, DCAKeep3rJobABI, args.fork.ethersProvider);
+
+  try {
+    const [pairs, intervals] = await job.connect(args.keeperAddress).callStatic.workable({
+      blockTag: args.advancedBlock,
+    });
+
+    logConsole.warn(`Job ${pairs.length ? 'is' : 'is not'} workable`);
+    if (!pairs.length) return args.subject.complete();
+
+    try {
+      await job.connect(args.keeperAddress).callStatic.work(pairs, intervals, {
+        blockTag: args.advancedBlock,
+      });
+    } catch (err: any) {
+      logConsole.warn('Workable but failed to work', {
+        message: err.message,
+      });
+      return args.subject.complete();
+    }
+
+    const workableGroups: JobWorkableGroup[] = [];
+
+    for (let index = 0; index < args.bundleBurst; index++) {
+      const tx = await job.connect(args.keeperAddress).populateTransaction.work(pairs, intervals, {
```

Comment:
I believe we can eliminate the for loop and place the tx outside the loop as it will always be the same tx. job-strategies in sample-jobs is a good example to follow. It also has some brief comments to orientate the reader, we can add them too, but please confirm with the gorilla